src/js/video.js

```diff
@@ -42,6 +43,11 @@ videojs.players = Player.players;
 
 videojs.extends = extendsFn;
 
+videojs.getGlobalOptions = () => options;
```

Comment:
If we're going to start using fat arrows, make sure to make a note in the ES6 page on the wiki.